contributed_definitions/NXreflections.nxdl.xml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+  # NXreflections: candidate NeXus definition for reflection data
+-->
+<definition type="group" name="NXreflections" category="contributed" extends="NXobject"
+    xmlns="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@ ../nxdl.xsd"
+    version="1.0"
+    >
+  <doc> This is a definition for reflection data from diffraction experiments </doc>
+  <group type="NXentry" minOccurs="1">
+    
+    <field name="definition">
+      <doc> NeXus NXDL schema to which this file conforms </doc>
+      <enumeration>
+        <item value="NXreflections" />
+      </enumeration>
+    </field>
+
```

Comment:
lxml.etree.DocumentInvalid: Element '{http://definition.nexusformat.org/nxdl/3.1}field', attribute 'type': 
    [facet 'enumeration'] The value 'NXmx' is not an element of the set
        {'NX_CHAR', 'NX_FLOAT', 'NX_INT', 'NX_UINT', 'NX_POSINT', 'NX_NUMBER', 
        'NX_DATE_TIME', 'ISO8601', 'NX_BOOLEAN', 'NX_BINARY'}., 
    line 22


Suggest removing ``type="NXmx"`` (defaults to NX_CHAR)